JavaArchitecturesAndDesignPatterns/patterns/behavioral/src/main/java/command/User.java

```diff
@@ -0,0 +1,25 @@
+package command;
+
+public class User {
+    final ReportI genClientsRep;
+    final ReportI genAccountingRep;
+    final ReportI genGoodsRep;
+
+    public User(ReportI genClientsRep, ReportI genAccountingRep, ReportI genGoogsRep) {
+        this.genClientsRep = genClientsRep;
+        this.genAccountingRep = genAccountingRep;
+        this.genGoodsRep = genGoogsRep;
+    }
+
+    void genClientsRep() {
+        genClientsRep.Generate();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.